src/malli/util.cljc

```diff
@@ -408,14 +408,18 @@
     (-children-schema [_ _])
     (-into-schema [parent properties children options]
       (m/-check-children! type properties children min max)
-      (let [[children forms schema] (fn properties (vec children) options)
+      (let [[children forms schema] (func properties (vec children) options)
             form (delay (m/-create-form type properties forms options))
             cache (m/-create-cache options)]
         ^{:type ::m/schema}
         (reify
           m/Schema
           (-validator [_] (m/-validator schema))
           (-explainer [_ path] (m/-explainer schema path))
+          (-parser [this]
```

Comment:
shoudn't these be forwarded to the schema, e.g. `(m/parser schema)` and `(m/unparser schema)`?

Reply:
Yep, you are right, I already figured it out when was playing with new version. Updated implementation accordingly. Thank you!